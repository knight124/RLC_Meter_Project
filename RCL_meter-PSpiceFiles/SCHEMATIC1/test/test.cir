** Profile: "SCHEMATIC1-test"  [ F:\RLC_METER_PROJECT\RCL_meter-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../rcl_meter-pspicefiles/rcl_meter.lib" 
* From [PSPICE NETLIST] section of F:\cadence\Cadence\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
