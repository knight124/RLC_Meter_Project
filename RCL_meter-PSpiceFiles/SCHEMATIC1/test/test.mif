 lib=F:\RLC_METER_PROJECT\rcl_meter-pspicefiles\rcl_meter.lib, offset=42, size=180
 lib=F:\RLC_METER_PROJECT\rcl_meter-pspicefiles\rcl_meter.lib, offset=223, size=390
