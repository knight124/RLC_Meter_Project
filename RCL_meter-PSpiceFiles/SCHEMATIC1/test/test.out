
**** 10/05/19 20:01:53 ****** PSpice 17.2.0 (March 2016) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ F:\RLC_METER_PROJECT\RCL_meter-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../rcl_meter-pspicefiles/rcl_meter.lib" 
* From [PSPICE NETLIST] section of F:\cadence\Cadence\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source RCL_METER
X_X1         0 N03235 QZS10MEG
R_R1         0 N03235  120k TC=0,0 
R_R2         N03235 N03440  90k TC=0,0 
C_C1         N02987 N03235  180p  TC=0,0 
C_C2         0 N02987  180p  TC=0,0 
R_R3         0 N02987  6k TC=0,0 
Q_Q1         N03440 N03235 N02987 Qbreakn 
V_V1         N03440 0 12

**** RESUMING test.cir ****
.END

**** 10/05/19 20:01:53 ****** PSpice 17.2.0 (March 2016) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ F:\RLC_METER_PROJECT\RCL_meter-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               Qbreakn         
               NPN             
       LEVEL    1            
          IS  100.000000E-18 
          BF  200            
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE   15.000000E-12 
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.42         
           D     .87         


**** 10/05/19 20:01:53 ****** PSpice 17.2.0 (March 2016) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ F:\RLC_METER_PROJECT\RCL_meter-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N02987)    5.8350 (N03235)    6.6083 (N03440)   12.0000 (X_X1.3)    6.6083     

(X_X1.11)    0.0000                   (X_X1.12)    6.6083                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1        -1.028E-03
    X_X1.vsin    0.000E+00

    TOTAL POWER DISSIPATION   1.23E-02  WATTS



          JOB CONCLUDED

**** 10/05/19 20:01:53 ****** PSpice 17.2.0 (March 2016) ****** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ F:\RLC_METER_PROJECT\RCL_meter-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .02
